.github/workflows/pr-description-update.yml

```diff
@@ -1,50 +0,0 @@
-name: "Update Pull Request Description with conventional commits"
```

Comment:
how come youre removing this?

Reply:
We don't use this workflow on our main repo anymore: `https://github.com/kurtosis-tech/kurtosis/tree/main/.github/workflows`. Engineers decided it wasn't useful to document descriptions with conventional commits a while back.